examples/data_pipeline/transforms.py

```diff
@@ -57,9 +57,9 @@ def __init__(self, vocab):
     def forward(self, tokens: List[str]) -> List[int]:
         return self.vocab.lookup_indices_1d(tokens)
 
-    def to_ivalue(self):
-        if hasattr(self.vocab, 'to_ivalue'):
-            vocab = self.vocab.to_ivalue()
+    def __prepare_scriptable__(self):
+        if hasattr(self.vocab, '__prepare_scriptable__'):
```

Comment:
Same here

Reply:
This is not necessary any more since the `__prepare_scriptable__` func recursively prepares the components.